platform/ATCmdParser.h

```diff
@@ -271,6 +271,9 @@ class ATCmdParser : private NonCopyable<ATCmdParser> {
 
     /**
      * Direct scanf on underlying stream
+    * This function does not itself match whitespace in its format string, so \n is not significant to it.
```

Comment:
Please fix indentation. I would say that ESP8266 and ATCmdParser changes should be done on separate commits.